Task

ahmedkashwaa opened the pull request:
test

ahmedkashwaa committed:
Task

mariam45677 commented:
اولا اول تاسك  ي احمد عاوزة اعرف رن معاك اصلا لان مطلعلي ايرور بلس كمان ليه عامل كلاس غير السيلندر والسيركل وعامل الماغيرات فيه كل الحكاية كانت هتخلص ف كلاس واحد 
ثانيا مستخدمتش ولا ميثود من اللي عاوزاهم كان ممكن تعمل ميثود لمساحة الدايرة وتعملها override ف كلاس تاني

بالنسبة لتاني تاسك كل الكود اللي انت كاتبه لما تيجي تعمل try  اكتب كودك جواها تمام؟

mariam45677 committed:
Update test.java

كومنت بسيط دخل كل كودك اللي عاوز تتحقق منه جوه ال try 
زي ماعملتهولك كدا

mariam45677 committed:
Update test.java

اول مسالة رنت معاك؟ مبترنش معايا بلس مش فاهمه اصلا كل الحكاية عاوزة تستخدم ميثود واتنين كلاس اليركل والسيلندر فقط والجيتر والسيتر كانت هتسهل عليك دول

mariam45677 committed:
Update test.java
